content/blog/2018-11-27-react-16-roadmap.md

```diff
@@ -58,153 +56,148 @@ function MyComponent() {
+**Statut dans React DOM Server :** les mêmes versions 16.7 alpha de `react-dom` prennent pleinement les Hooks en charge avec `react-dom/server`.
 
-**Status in React Native:** There is no officially supported way to try Hooks in React Native yet. If you're feeling adventurous, check out [this thread](https://github.com/facebook/react-native/issues/21967) for unofficial instructions. There is a known issue with `useEffect` firing too late which still needs to be solved.
+**Statut dans React Native :** React Native ne fournit pas pour le moment de prise en charge officielle des Hooks.  Mais si vous êtes joueur·se, jetez un coup d’œil à [cette discussion](https://github.com/facebook/react-native/issues/21967) pour des instructions non-officielles.  Notez qu’un problème identifié de déclenchement tardif de `useEffect` n’est pas encore résolu.
 
-**Recommendation:** When you’re ready, we encourage you to start trying Hooks in new components you write. Make sure everyone on your team is on board with using them and familiar with this documentation. We don’t recommend rewriting your existing classes to Hooks unless you planned to rewrite them anyway (e.g. to fix bugs). Read more about the adoption strategy [here](/docs/hooks-faq.html#adoption-strategy).
+**Notre conseil :** quand vous serez prêt·e, nous vous encourageons à essayer les Hooks dans les nouveaux composants que vous écrirez.  Assurez-vous que tous les membres de votre équipe sont partant·e·s et ont parcouru la documentation associée.  Nous déconseillons de réécrire vos classes existantes vers les Hooks à moins que vous n’ayez déjà prévu de les réécrire pour d’autres raisons (par ex. pour corriger des bugs).  Vous pouvez en apprendre davantage sur la stratégie d’adoption [ici](/docs/hooks-faq.html#adoption-strategy).
 
-### React 16.x (~Q2 2019): The One with Concurrent Mode {#react-16x-q2-2019-the-one-with-concurrent-mode}
+### React 16.x (~Q2 2019) : celle avec le mode concurrent {#react-16x-q2-2019-the-one-with-concurrent-mode}
 
-*Concurrent Mode* lets React apps be more responsive by rendering component trees without blocking the main thread. It is opt-in and allows React to interrupt a long-running render (for example, rendering a new feed story) to handle a high-priority event (for example, text input or hover). Concurrent Mode also improves the user experience of Suspense by skipping unnecessary loading states on fast connections.
+Le *mode concurrent* permet aux applis React d’être plus réactives en réalisant le rendu des arbres de composants sans bloquer le thread principal.  Il doit être explicitement activé et permet à React d’interrompre un rendu long (par exemple, le rendu d’un long fil de discussion) afin de traiter des événements de plus forte priorité (tels qu’une saisie clavier ou un survol de la souris).  Le mode concurrent améliore également l’expérience utilisateur de Suspense en sautant les états de chargement superflus lorsque la connexion est rapide.
```

Comment:
"sans bloquer le thread principal"

- soit on traduit  "thread"

-  soit on le met entre "*" => "sans bloquer le \*thread\* principal"

Ton avis ?

Reply:
Alors justement :

1. La trad « officielle » française (fil) n'est pas établie, c'est-à-dire qu'elle n’est jamais utilisée (un peu comme « bogue », mais pire). C'est la VO qui est établie, comme « commit » dans l'univers de la gestion de source.
2. Partout ailleurs dans les docs, on garde “thread” sans italiques, sauf erreur de ma part, comme pour “commit” et “bundle”, notamment.

Reply:
Ok c'est noté